_scripts/banned_plugins.js

```diff
@@ -2,5 +2,7 @@ module.exports = [
 
   'chai-51st-state', // Banned for profane language
   'chai2.0-webdriver-promised', // Fork with poor metadata
+  'lodash-match-pattern', // not a real chai plugin, chai-match-pattern is the real one
```

Comment:
Opened https://github.com/Originate/lodash-match-pattern/pull/7 for this. Let's see if they remove quickly so we can avoid this.